tests/test_multipart.py

```diff
@@ -546,6 +546,21 @@ async def test_read_form(self) -> None:
             result = await obj.form()
         assert [("foo", "bar"), ("foo", "baz"), ("boo", "")] == result
 
+    async def test_read_form_invalid_utf8(self) -> None:
+        invalid_unicode_byte = b"\xff"
+        data = invalid_unicode_byte + b"%s--:--" % newline
+        with Stream(data) as stream:
+            obj = aiohttp.BodyPartReader(
+                BOUNDARY,
+                {CONTENT_TYPE: "application/x-www-form-urlencoded"},
+                stream,
+                _newline=newline,
```

Comment:
Test are failing because this argument doesn't exist in 3.9 (looks like it didn't get backported from master).

Reply:
Let's see now!